q2_fondue/sequences.py

```diff
@@ -357,11 +367,17 @@ def get_sequences(
     processed_q = manager.Queue()
 
     with tempfile.TemporaryDirectory() as tmp_dir:
+        # get dbGAP key for restricted access sequences
+        if restricted_access:
+            dotenv.load_dotenv()
+            key_file = os.getenv('KEY_FILEPATH')
```

Comment:
I would add here a check that the file actually exists (and raise an error if it doesn't). You could e.g. use something like `os.path.isfile().